mobile/lib/src/infrastructure/eeg_data_transformers/cytonTransformer.dart

```diff
@@ -0,0 +1,79 @@
+import 'dart:math';
+
+import 'package:polydodo/src/infrastructure/constants.dart';
+import 'baseTransformer.dart';
+
+class CytonTransformer<S, T> extends BaseTransformer<S, T> {
```

Comment:
Optionnel, unit test this class and ganglion

Reply:
Added Unit tests, not sure I'm liking the way it's done tho, Open to feedback.